### DISCUSSION TITLE
What is the best tank class on here?

### DISCUSSION THREAD

**Discussion by hjortmar** [chromiecraft/chromiecraft#3147] [General]
Pally viable? Druid okay still? Warrior top dawg?

**Comment by Annamaria-CC** [comment]
Hmm. For progression either a Druid or warrior works best. Druid has insane health pool which is forgiving on your healers. Warrior with their insane defensive cooldowns is also very nice to have. I have seen raids use paladins as tanks too, but it seems to be less used at the moment :)

**Reply by hjortmar** [reply]
So its alot like vanilla? :P Are pallies viable or are they still missing stuff?

**Reply by Annamaria-CC** [reply]
They are definitely viable, just less optimal.
